### PR TITLE
java sol for powered addition

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/1338/submission/278348871

File: solutions/silver/cf-1338A.mdx [hunk 1]
```diff
@@ -65,6 +65,36 @@ int main() {
...
+
+<JavaSection>
+```java
+import java.util.*;
+
+public class PoweredAddition {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int t = sc.nextInt();
+		while (t-- > 0) {
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1338A.mdx:77]: rename to `testNum` and use a for loop

File: solutions/silver/cf-1338A.mdx [hunk 2]
```diff
@@ -65,6 +65,36 @@ int main() {
...
+			int n = sc.nextInt();
+			int Arr[] = new int[n];
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1338A.mdx:79]: rename to `arr`

File: solutions/silver/cf-1338A.mdx [hunk 3]
```diff
@@ -65,6 +65,36 @@ int main() {
...
+			for (int i = 0; i < n; i++) { Arr[i] = sc.nextInt(); }
+
+			int prev = Arr[0];
+			int res = 0;
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1338A.mdx:83]: yeah uh can you make this consistent w/ the c++ version
honestly i feel like this is better suited for a user solution

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/silver/cf-1338A.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/silver/cf-1338A.mdx]: done

File: solutions/silver/cf-1338A.mdx (hunk 2 above)
Comment (freakin23) [reply solutions/silver/cf-1338A.mdx]: done.

File: solutions/silver/cf-1338A.mdx (hunk 3 above)
Comment (freakin23) [reply solutions/silver/cf-1338A.mdx]: okay

File: solutions/silver/cf-1338A.mdx [hunk 4]
```diff
@@ -65,6 +65,45 @@ int main() {
...
 </CPPSection>
+
+<JavaSection>
+```java
+import java.util.*;
+
+public class PoweredAddition {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int testNum = sc.nextInt();
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1338A.mdx:76]: i think 99% of java sols use bufferedreader

Comment (freakin23) [reply solutions/silver/cf-1338A.mdx]: ok

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/1338/submission/278422810

**Pull request merged**